common/views/components/Pagination/Pagination.tsx

```diff
@@ -151,15 +149,14 @@ export const Pagination: FunctionComponent<Props> = ({
       {isEnhanced ? (
         <>
           <span aria-hidden>Showing page</span>
-          <span id="searchInputLabel" className="visually-hidden">
-            {`Showing page ${currentPage} / ${formatNumber(totalPages)}`}
-          </span>
           <PageSelectorInput
             name="page"
             onFocus={() => setIsFocused(true)}
             onBlur={() => setIsFocused(false)}
             form={isFocused ? 'search-page-form' : ''}
-            aria-labelledby="searchInputLabel"
+            aria-label={`Showing page ${currentPage} / ${formatNumber(
+              totalPages
+            )}`}
```

Comment:
@gestchild (for the benefit of a thread)
I hadn't though about reviewing the content, good idea!

The VoiceReader does announce that it's an input field/"edit text", but maybe the "Showing page" is slightly confusing? And agree, being read "slash", better off with "of"
<img width="670" alt="Screenshot 2023-06-02 at 11 33 15" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/110461050/b4fd0c37-815c-42c8-bfd7-71685a710c97">

If went down a rabbit hole and considered `role="range"` with `aria-maxvalue` and `aria-minvalue` but apparently it's not best practice 🤷‍♀️ 

Would something like: "Jump to page [currentPage] of [total number]" work best?

Reply:
I think that is an improvement yeah